### ISSUE TITLE
loading aliases from tsconfig.json?

### ISSUE THREAD

**Issue by jacekkopecky** [zachbryant/parcel-resolver-tspaths#5]
In Typescript, we can do path aliases in tsconfig.json, such as 

```json
    "baseUrl": "./",
    "paths": {
      "shared/*": ["pages/shared/*"]
    }
```

which are taken into account for typescript import like this:

```typescript
import Icon from 'shared/components/Icon';
```

Parcel doesn't seem to take these into account when bundling, instead it expects a copy of the alias in `package.json`:

```json
  "alias": {
    "shared": "./pages/shared"
  }
```

Example code that shows this is in https://github.com/jacekkopecky/parcel-alias-import-example 

**I'd expect `parcel-resolver-tspaths` to be able to remove the need for `package.json` aliases here (please correct me if there are reasons against this).**

An interesting extra consideration is that `tsconfig.json` paths can start with `@` (as used in many examples online) so the import could be `'@shared/components/Icon'` but `package.json` aliases seem to break with an `@`.

**Comment by zachbryant** [comment]
Hey so I ran your example code, and identified a bug in accessing the configured baseUrl. Thanks for reporting and supplying the example repo. I've published the fix in `0.0.8`

**Comment by jacekkopecky** [comment]
Great, thanks! It works with `@`, too!
